core/liquidity-manager/main.ts

```diff
@@ -29,7 +26,7 @@ class App {
   async run(): Promise<void> {
     await this.node.init();
 
-    while (true) {
+    for (;;) {
```

Comment:
whats the difference between those?

Reply:
Linter does not allow a constant in a while loop 🤣